test/spec/validations/missing_start_date.js

```diff
@@ -0,0 +1,80 @@
+describe('iD.validations.missing_start_date', function () {
+    var context;
+
+    beforeEach(function() {
+        context = iD.coreContext().assetPath('../dist/').init();
+    });
+
+    function createWay(tags) {
+        var n1 = iD.osmNode({id: 'n-1', loc: [4,4]});
+        var n2 = iD.osmNode({id: 'n-2', loc: [4,5]});
+        var n3 = iD.osmNode({id: 'n-3', loc: [5,5]});
+        var w = iD.osmWay({id: 'w-1', nodes: ['n-1', 'n-2', 'n-3'], tags: tags});
+
+        context.perform(
+            iD.actionAddEntity(n1),
+            iD.actionAddEntity(n2),
+            iD.actionAddEntity(n3),
+            iD.actionAddEntity(w)
+        );
+    }
+
+    function validate() {
+        var validator = iD.validationMissingStartDate(context);
+        var changes = context.history().changes();
+        var entities = changes.modified.concat(changes.created);
+        var issues = [];
+        entities.forEach(function(entity) {
+            issues = issues.concat(validator(entity, context.graph()));
+        });
+        return issues;
+    }
+
+    it('has only missing tag errors on init', function() {
+        var issues = validate();
+        expect(issues).to.have.lengthOf(0);
+    });
+
+    it('ignores way with no tags', function() {
+        createWay({});
+        var issues = validate();
+        expect(issues).to.have.lengthOf(0);
+    });
+
+    it('ignores way with start_date tag', function() {
+        createWay({ start_date: '1950' });
+        var issues = validate();
+        expect(issues).to.have.lengthOf(0);
+    });
+
+    it('ignores way with start_date:edtf tag', function() {
+        createWay({ 'start_date:edtf': '1950' });
```

Comment:
Eventually, we may decide to warn about a missing `start_date` in this case as well, since the map renderer doesn’t know anything about `start_date:edtf` yet. Typically, we’ve been considering `start_date:edtf` to be a complement to `start_date` rather than a replacement for it. However, I think what you have currently is correct. I also made the same assumption in #188: the suggested fix for that warning only sets `start_date:edtf` but not `start_date`.